yarn.lock

```diff
@@ -30953,7 +30953,7 @@ string-replace-loader@^3.1.0:
     loader-utils "^2.0.0"
     schema-utils "^3.0.0"
 
-"string-width-cjs@npm:string-width@^4.2.0":
+"string-width-cjs@npm:string-width@^4.2.0", string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
```

Comment:
Why there are changes to yarn.lock?

Reply:
I think it's possible someone didn't merge their yarn.lock in a PR that went into main, cause these are the changes that are being generated via `yarn kbn bootstrap`.